ambry-store/src/main/java/com.github.ambry.store/Offset.java

```diff
@@ -36,11 +36,11 @@
    * Construct an Offset that refers to a position in Log.
    * @param name the name of segment being referred to.
    * @param offset the offset within the segment.
-   * @throws IllegalArgumentException if {@code name} is {@code null} or empty or {@code offset} < 0.
+   * @throws IllegalArgumentException if {@code name} is {@code null} or {@code offset} < 0.
    */
   Offset(String name, long offset) {
```

Comment:
I understand the intent but I don't think `offset.getOffsetInto()` or `offset.getLogOffset()` sounds any better. @pnarayanan pls comment.